functions/src/index.ts

```diff
@@ -0,0 +1,58 @@
+// Start writing Firebase Functions
+// https://firebase.google.com/docs/functions/typescript
+
+'use strict';
+
+const functions = require('firebase-functions');
+const admin = require('firebase-admin');
+admin.initializeApp();
+
+const https = require('https');
+const uuidv1 = require('uuid/v1');
+
+exports.createCharge = functions.https.onCall((data, context) => {
+  // TODO (@missmaggiemo): Replace with MCS access token
+  const SANDBOX_ACCESS_TOKEN = 'EAAAEPhzZ1_bxxI-zoFHW5RY_ESLZdimRPa0jy9DGxqbBw1NSUMcYY3OFLp-3N5P';
```

Comment:
We'll have to be careful that we don't check the keys into the repo 😅... It's been on my todo list to break out keys into a config file or env vars. Want to put some thought into this yourself and we can talk about it at the next hack day?